### PR TITLE
CONTRIB: Add script to upload ucx.pdf to wiki.

### LINKED ISSUE #779: Prepare a script for automatic generation / upload of doxygen

### PR DESCRIPTION
**Pull request by yosefe**
Fixes #779

### CODE REVIEW

**Comment by yosefe** [comment]
@shamisp pls review

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1965/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1340/ for details (Mellanox internal link).

File: contrib/upload_docs.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -eEx
+#
+# Run this from UCX build directory
+#
+
+make docs
+mkdir -p wcx.wiki
```
Comment (alinask) [inline contrib/upload_docs.sh]: ucx.wiki ?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1344/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1969/ for details.

**Comment by shamisp** [comment]
@yosefe - please add copyright.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1355/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1979/ for details.

**Comment by yosefe** [comment]
@shamisp fixed

**Comment by shamisp** [comment]
👍

**Pull request merged**